static/mmci-cli/models/NK_DEFK17/NK_DEFK17.json

```diff
@@ -0,0 +1,327 @@
+      "text": "N"
+    },
+    {
+      "name": "inflation",
+      "text": "inflation"
+    },
+    {
+      "name": "inflationq",
+      "text": "inflationq"
+    },
+    {
+      "name": "interest",
+      "text": "interest"
+    },
+    {
+      "name": "Ng",
+      "text": "Ng"
+    },
+    {
+      "name": "LY",
+      "text": "LY"
+    },
+    {
+      "name": "QK",
+      "text": "QK"
+    },
+    {
+      "name": "GDP",
+      "text": "GDP"
+    },
+    {
+      "name": "output",
+      "text": "output"
+    },
+    {
+      "name": "outputgap",
+      "text": "outputgap"
+    },
+    {
+      "name": "R",
+      "text": "R"
+    },
+    {
+      "name": "Q",
+      "text": "Q"
+    },
+    {
+      "name": "pI",
+      "text": "pI"
+    },
+    {
+      "name": "RK",
+      "text": "RK"
+    },
+    {
+      "name": "rK",
+      "text": "rK"
+    },
+    {
+      "name": "rr",
+      "text": "rr"
+    },
+    {
+      "name": "ERQ",
+      "text": "ERQ"
+    },
+    {
+      "name": "w",
+      "text": "w"
+    },
+    {
+      "name": "infl_w",
+      "text": "infl_w"
+    },
+    {
+      "name": "X1w",
+      "text": "X1w"
+    },
+    {
+      "name": "X2w",
+      "text": "X2w"
+    },
+    {
+      "name": "mc",
+      "text": "mc"
+    },
+    {
+      "name": "infl",
+      "text": "infl"
+    },
+    {
+      "name": "X1p",
+      "text": "X1p"
+    },
+    {
+      "name": "X2p",
+      "text": "X2p"
+    },
+    {
+      "name": "Delta_p",
+      "text": "Delta_p"
+    },
+    {
+      "name": "CY",
+      "text": "CY"
+    },
+    {
+      "name": "Spr",
+      "text": "Spr"
+    },
+    {
+      "name": "phi",
+      "text": "phi"
+    },
+    {
+      "name": "C_f",
+      "text": "C_f"
+    },
+    {
+      "name": "Inv_f",
+      "text": "Inv_f"
+    },
+    {
+      "name": "S_Inv_f",
+      "text": "S_Inv_f"
+    },
+    {
+      "name": "dS_Inv_f",
+      "text": "dS_Inv_f"
+    },
+    {
+      "name": "H_f",
+      "text": "H_f"
+    },
+    {
+      "name": "Y_f",
+      "text": "Y_f"
+    },
+    {
+      "name": "tau_f",
+      "text": "tau_f"
+    },
+    {
+      "name": "K_f",
+      "text": "K_f"
+    },
+    {
+      "name": "N_f",
+      "text": "N_f"
+    },
+    {
+      "name": "Ng_f",
+      "text": "Ng_f"
+    },
+    {
+      "name": "LY_f",
+      "text": "LY_f"
+    },
+    {
+      "name": "QK_f",
+      "text": "QK_f"
+    },
+    {
+      "name": "GDP_f",
+      "text": "GDP_f"
+    },
+    {
+      "name": "R_f",
+      "text": "R_f"
+    },
+    {
+      "name": "R0_f",
+      "text": "R0_f"
+    },    
+    {
+      "name": "Q_f",
+      "text": "Q_f"
+    },
+    {
+      "name": "pI_f",
+      "text": "pI_f"
+    },
+    {
+      "name": "RK_f",
+      "text": "RK_f"
+    },
+    {
+      "name": "rK_f",
+      "text": "rK_f"
+    },
+    {
+      "name": "ERQ_f",
+      "text": "ERQ_f"
+    },
+    {
+      "name": "w_f",
+      "text": "w_f"
+    },
+    {
+      "name": "infl_w",
```

Comment:
this is a duplicate of line 179

Reply:
Is that a problem?

Reply:
Otherwise I'd merge the PR...

Reply:
yes, it will lead to odd behavior of the variable seelction list.

Reply:
Sorry for that, must have happened when I added the flex-price variables. I'll correct that and send a new PR!